src/index.css

```diff
@@ -144,3 +144,20 @@ textarea {
   overflow: auto;
   overflow-y: hidden;
 }
+
```

Comment:
There shouldn't need to be any css changes for this. Some of these classes are overwriting classes from other css files and having unintended side effects

Reply:
Hi aubundy. I didn't think I changed anything in index.css at line 144. Should the overflow and overflow-y lines not be there?

Reply:
Line 147 was added (note the green highlight with "+"). Maybe your code formatter did this automatically?